src/goPath.ts

```diff
@@ -41,6 +41,15 @@ export function getBinPathWithPreferredGopath(toolName: string, preferredGopaths
 	}
 
 	const binname = alternateTool && !path.isAbsolute(alternateTool) ? alternateTool : toolName;
+	const env = Object.assign({}, process.env);
```

Comment:
What is the reason for making a copy of the env variables?
Since we are only reading the env var values, `process.env` can be directly used.